libs/user/src/lib/user.pipe.ts

```diff
@@ -10,3 +10,5 @@ export class UserPipe implements PipeTransform {
     return `${user.name} ${user.lastname} - ${user.country}`;
   }
 }
+
+// https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin/docs/rules/no-host-metadata-property.md
```

Comment:
I don't get it. I don't have any eslint error, did you try to restart your linter ?

Reply:
Didn't help.  There is probably a misconfiguration somewhere.

Reply:
This is weird indeed